### PR TITLE
GLB-487-459-Most-Popular-Section-and-Filter-by-Category

### PR DESCRIPTION
**Pull request by MZegarraGlobant**
Why are we making this change?
- The user is able to filter posts in blog sections.
- The user is able to see the most popular section.

How did you implement the changes?
- We renamed the `BrowseAll` component to `PostPreviewsGrid` because this component can handle more sections on the page thanks to its dynamic structure.
- We reuse the `PostPreviewsGrid` to create the Most Popular section.

# UI changes?

Add screenshots
GLB-459

### CODE REVIEW

File: components/EditorialStory/index.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ interface IEditoralStory {
...
 const EditoralStory = ({ posts }: IEditoralStory) => {
   const [firstPost, ...columnPosts] = posts;
+  const secondaryPost = columnPosts.slice(1, 4);
   return (
     <>
       <div className="editorialStory">
         <PostPreview post={firstPost} variant="front" />
         <div className="columnPosts">
-          {columnPosts.map((element, key) => (
+          {secondaryPost.map((element, key) => (
```
Comment (g-azambrano) [inline components/EditorialStory/index.tsx:16]: Why was this change needed? 🤔

File: pages/index.tsx [hunk 2]
```diff
@@ -39,7 +39,11 @@ const Home: NextPage<IHomeProps> = ({ posts }) => {
         posts={posts}
         categoryFilter
       />
-      {/* <BlogSection title="BROWSE ALL" variant="grid" posts={posts} /> */}
+      <BlogSection
+        title="MOST POPULAR"
+        variant={BlogSectionVariants.mostPopular}
+        posts={posts}
+      />
```
Comment (g-azambrano) [inline pages/index.tsx]: Let's just remember that this section, Most Popular, should be visible before the Browse All section.

Right now, they are inverted.

**Review by g-azambrano — COMMENTED** [review]
The final product is looking good! Please revert the order of the Blog sections and that should be it for this task.

Let's discuss other items to improve at the code level, but that could be at the end when every task is fulfilled.

File: components/EditorialStory/index.tsx (hunk 1 above)
Comment (MZegarraGlobant) [reply components/EditorialStory/index.tsx:16]: Because of the naming and the `secondaryPost` on the component only should display the 3 posts at the time.

**Review by g-azambrano — APPROVED** [review]
👍
